flake.nix

```diff
@@ -0,0 +1,27 @@
+{
+  description = "My haskell application";
```

Comment:
not sure if the description is correct?

Reply:
Oops. Changed it to "Shell script collection to download ELF binaries and use them right away!".